resource-server-utils/src/main/java/org/jasig/resourceserver/utils/taglib/JavaScriptMinificationTag.java

```diff
@@ -32,9 +32,9 @@
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.jasig.resource.aggr.CommonsLogErrorReporter;
-import org.jasig.resource.com.yahoo.platform.yui.compressor.JavaScriptCompressor;
-import org.jasig.resource.org.mozilla.javascript.ErrorReporter;
-import org.jasig.resource.org.mozilla.javascript.EvaluatorException;
+import com.yahoo.platform.yui.compressor.JavaScriptCompressor;
+import org.mozilla.javascript.ErrorReporter;
+import org.mozilla.javascript.EvaluatorException;
```

Comment:
This could cause issues, a similar change had to be reverted because it blocked the release build. https://github.com/Jasig/resource-server/commit/00c10f26d3664662b75b337497a2afe9450e72c3
/cc @drewwills for historical context on `org.jasig.resource.org` prefix